runhouse/servers/obj_store.py

```diff
@@ -90,15 +99,6 @@ def initialize(
                 "Must provide a servlet name if the servlet has local storage."
             )
 
-        if (
```

Comment:
Removing this for now. You can initialize an `obj_store` with a servlet that doesn't exist, it'll just throw an error later